bootstrap.sh

```diff
@@ -1,13 +1,18 @@
 #!/bin/sh
 
-if [ -z "$SCHEME" ]
+set -e  # Exit on any error
+
+echo "bootstrapping SCHEME=$SCHEME IDRIS2_VERSION=$IDRIS2_VERSION"
+if [ -z "$SCHEME" ] || [ -z "$IDRIS2_VERSION" ]
 then
-    echo "SCHEME not set. Invoke with SCHEME=[name of chez executable]"
+    echo "Required ENV not set."
```

Comment:
I expect forgetting to set `SCHEME` will be a common error (I've done it myself many times) so could the error message still report specifically if `SCHEME` is missing? I guess it'll get `IDRIS2_VERSION` from the Makefile.

Reply:
Actually I'm just going to merge then update this I think.

Reply:
Thanks. I think in the future we should actually have a configure stage that searches for a suitable scheme without the user having to provide it manually.